.env.example

```
@@ -1,19 +1,8 @@
-
-# Site Descriptions (Multilingual) - EN before ES
-SITE_DESCRIPTION_EN="Portfolio of Yuniel Acosta, Full Stack Developer"
-SITE_DESCRIPTION_ES="Portafolio de Yuniel Acosta, Desarrollador Full Stack"
-
-# Social Media
-X_ACCOUNT=@yacosta738
-
-# Translation Messages - EN before ES
-NOT_TRANSLATED_CAUTION_EN="This page is not available in your language."
-NOT_TRANSLATED_CAUTION_ES="Esta página no está disponible en tu idioma."
+DOMAIN="yunielacosta.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)